scc/cert/serialization/types_test.go

```diff
@@ -0,0 +1,66 @@
+package serialization
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestHexBytes_UnmarshalJSON_ValidHexString(t *testing.T) {
+	var h HexBytes
+	data := []byte(`"0x012abc"`)
+	require.NoError(t, h.UnmarshalJSON(data))
+	data = []byte(`"null"`)
+	require.NoError(t, h.UnmarshalJSON(data))
+}
```

Comment:
This test misses a case with an odd number of hex-characters.

Reply:
test case added